### ISSUE TITLE
env var & fieldRef bug

### ISSUE THREAD

**Issue by vbehar** [arttor/helmify#74]
Hi,

https://github.com/arttor/helmify/pull/72 released in https://github.com/arttor/helmify/releases/tag/v0.3.24 introduced a bug in the env vars using a `fieldRef`, such as

```yaml
        - name: OPERATOR_NAMESPACE
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
```

which gets converted to

```yaml
        - name: OPERATOR_NAMESPACE
          value: {{ .Values.controllerManager.manager.operatorNamespace }}
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
```

because we don't handle `fieldRef` and `resourceFieldRef` - see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#envvarsource-v1-core

I'll open a PR tomorrow to fix it

**Comment by arttor** [comment]
@vbehar thank you for your contribution! changes are available in [v0.3.25 release](https://github.com/arttor/helmify/releases/tag/v0.3.25).

**Comment by vbehar** [comment]
great, thanks for the quick review and merge!
